klarna-on-demand/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.klarna.ondemand">
 
+    <uses-permission android:name="android.permission.READ_PHONE_STATE" />
```

Comment:
maybe add an explanation why this permission is needed to users in the Readme? 
Just so developers can be clear why is this used - if they don't use it before in their app.